frontend/test/metabase/scenarios/collection_defaults.cy.spec.js

```diff
@@ -0,0 +1,148 @@
+import {
+  restore,
+  signInAsAdmin,
+  setupEmail,
+  signInAsNormalUser,
+} from "__support__/cypress";
+// Ported from initial_collection.e2e.spec.js
+
+// Z because the api lists them alphabetically by name, so it makes it easier to check
+const collection_name = "Z Collection";
+const sub_collection_name = "ZZ Sub-Collection";
+const pulse_name = "Test pulse";
+const dashboard_name = "Test Dashboard";
+
+function createPulse() {
+  cy.visit("/pulse/create");
+  cy.findByPlaceholderText("Important metrics").type(pulse_name);
+  cy.findByText("Select a question").click();
+  cy.findByText("Orders").click();
+  cy.findByPlaceholderText(
+    "Enter email addresses you'd like this data to go to",
+  )
+    .click()
+    .clear();
+  cy.contains("Bobby").click();
+  cy.findByText("To:").click();
+
+  cy.findByText("Robert Tableton").should("not.exist");
+  cy.findByText("Bobby Tables");
+  cy.findByText("Create pulse").click();
+}
+
+describe("scenarios > collection_defaults", () => {
+  before(() => {
+    // if (window.Cypress) {
+    //   event.preventDefault();
+    // }
+    restore();
+    cy.server();
+  });
+
+  describe("for admins", () => {
+    beforeEach(signInAsAdmin);
+
+    describe("a new collection", () => {
+      it("should be the parent collection", () => {
+        // Make new collection
+        cy.request("POST", "api/collection/", {
+          name: collection_name,
+          color: "#ff9a9a",
+          personal_owner_id: 1,
+        });
+
+        // Check that it has no parent
+        const length = 7;
+        cy.request("api/collection").then(response => {
+          expect(response.body).to.have.length(length);
+          expect(response.body[length - 1].name).to.equal(collection_name);
+          expect(response.body[length - 1].location).to.equal("/");
+        });
```

Comment:
Now that these are in cypress, maybe they should test the frontend a bit too? Currently they do an API test from the browser, but they could also test the creation modal if thats not happening elsewhere or test that it appears on `/collection/root` to assert the location.

Reply:
Sure, I'll add that in